added progress bars, components sectiontitle and section subtitle, ta…

echoreverb opened the pull request:
- добавлены прогресс бары;
- созданы компоненты SectionTitle, SectionSubtitle;

echoreverb committed:
added progress bars, components sectiontitle and section subtitle, tasks.txt

echoreverb committed:
corrected values of statistics data

echoreverb committed:
misprint removed

Kigrar commented on components/StatisticsCard.vue:
```diff
@@ -1,15 +1,33 @@
+  },
 };
 </script>
 
```
```suggestion
.statistics-card {
  display: flex;
  flex-direction: column;
  align-items: flex-end;
  align-content: space-between;
  width: 100%;
  min-height: 300px;
  max-width: 300px;
  border: 1px solid #efefef;
  padding: 20px;
}
```

Делаем ячейки flex контейнерами. И убираем position.

Kigrar commented on components/StatisticsCard.vue:
```diff
@@ -27,12 +45,12 @@ export default {
   font-size: 12px;
   line-height: 16px;
   position: absolute;
+  width: calc(100% - 20px * 2);
```
```suggestion

```

Удаляем ненужный position и калькуляцию ширины. (padding у нас уже задан)

Kigrar commented on components/StatisticsCard.vue:
```diff
@@ -27,12 +45,12 @@ export default {
   font-size: 12px;
```
```suggestion
  align-self: flex-start;
  font-size: 12px;
```

Пока все элементы контейнера жмутся ко дну, text будет липнуть к началу.

Kigrar commented on components/StatisticsCard.vue:
```diff
@@ -27,12 +45,12 @@ export default {
   width: 100%;
   height: 40px;
-  background: #f4f4f4;
   margin-top: 144px;
```
```suggestion
  margin-top: auto;
```

Kigrar reviewed (changes requested):
Flex вместо position и width.

echoreverb committed:
Update components/StatisticsCard.vue

ок!

Co-authored-by: Andrew Mishin <39715551+Kigrar@users.noreply.github.com>

echoreverb committed:
Update components/StatisticsCard.vue

Co-authored-by: Andrew Mishin <39715551+Kigrar@users.noreply.github.com>

echoreverb committed:
Update components/StatisticsCard.vue

Co-authored-by: Andrew Mishin <39715551+Kigrar@users.noreply.github.com>

echoreverb committed:
Update components/StatisticsCard.vue

Co-authored-by: Andrew Mishin <39715551+Kigrar@users.noreply.github.com>

echoreverb committed:
flex applied to options__column

Флекс блока с текстом и кнопкой, до этого кнопка уезжала вниз при выборе 2-го варианта

echoreverb committed:
responsive iframe

теперь видео сохраняет пропорции при изменении ширины

lagodinka commented on components/StatisticsCard.vue:
```diff
@@ -1,18 +1,46 @@
 };
 </script>
-
+.statistics-card {
```
вот тут стиль выбился из поля style

echoreverb replied:
исправлено

lagodinka reviewed (commented):
потерявшийся стиль для statistics-card

echoreverb committed:
Update StatisticsCard.vue

lagodinka reviewed (approved):
respect)
